src/agent/src/process_options_win.cpp

```diff
@@ -29,21 +23,9 @@ void StopAgent()
     WindowsService::ServiceStop();
 }
 
-void PrintHelp()
+void RestartAgent(const std::string& configFile)
```

Comment:
Since we are here, can we rename the `configFile` variable name to `configFilePath`?